app/src/main/res/values/styles.xml

```diff
@@ -53,7 +53,17 @@
         <item name="colorSecondary">@color/colorSecondary</item>
         <item name="colorOnSecondary">@color/white</item>
 
+        <item name="colorPrimaryContainer">@color/colorPrimary</item>
+        <item name="colorOnPrimaryContainer">@color/colorOnPrimary</item>
+
         <item name="colorSurface">@color/colorSurface</item>
+        <item name="colorSurfaceVariant">@color/colorSurface</item>
+
+        <item name="colorSurfaceContainerLowest">@color/colorSurface</item>
+        <item name="colorSurfaceContainerLow">@color/colorSurface</item>
+        <item name="colorSurfaceContainer">@color/colorSurface</item>
+        <item name="colorSurfaceContainerHigh">@color/colorSurface</item>
+        <item name="colorSurfaceContainerHighest">@color/colorSurface</item>
```

Comment:
These are supposed to indicate containers with different importance, but setting the accordingly looks weird or doesn't provide enough contrast.